liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -249,35 +250,29 @@ private String getPackageName() {
     }
 
     /**
-     * Returns canonical path to package directory
+     * Returns package directory
      * 
-     * @return canonical path to specified package directory, or default ${project.build.directory} (target) if unspecified
-     * @throws IOException
+     * @return File representing specified package directory, or default ${project.build.directory} (target) if unspecified
      */
-    private String getPackageDirectory() throws IOException, MojoFailureException {
+    private File getPackageDirectory() {
```

Comment:
For each of the returns in this method, I would return the canonical file, just like it previously returned the canonical path.